notebooks/02-Summary-statistics-and-plots.ipynb

```diff
@@ -0,0 +1,512 @@
+    "type(my_plots)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "313521d1-a033-4402-bb12-f495f4c76abd",
+   "metadata": {},
+   "source": [
+    "The object ~my_plots~ is a nested dictionary the \"keys\" to the dictionary are the names of the statistic that have been\n",
+    "plotted, and the values of these are themselves dictionaries of the types of plots that have been generated. We can\n",
+    "check what statistics were plotted and saved in a pickle by looking at the first level of keys."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1bf65589-e598-4176-913b-3924ee334c95",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_plots.keys()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "246fd31c-2b13-404b-9ef2-74e87ce5b0e7",
+   "metadata": {},
+   "source": [
+    "We see that the `area`, `area_cartesian_bbox` and `contour_lengths`, `end_to_end_distance` and `volume` statistics are\n",
+    "available for plotting. We can now look at what types of plots were generated by checking the keys for the dictionary\n",
+    "nested under this top-level (**NB** because of the way plots are generated all statistics will have the same plots\n",
+    "generated)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c8db9278-128d-48c1-9041-02eccd59f363",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_plots[\"area\"].keys()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d0d2ebe-081d-40a7-b7c0-4e72178937ec",
+   "metadata": {},
+   "source": [
+    "We see that both `dist` and `violin` plots were generated. To extract these to objects we unpack the values of the\n",
+    "dictionary. The following extracts the `figure` (the plot itself) and its `ax` (the \"axes\" which are the properties of\n",
+    "the plot) to their own objects."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b76c70eb-ef8e-4a26-8170-702b0b2bd809",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "figure_area_dist, ax_area_dist = my_plots[\"area\"][\"dist\"].values()\n",
+    "figure_area_violin, ax_area_violin = my_plots[\"area\"][\"violin\"].values()\n",
+    "figure_contour_lengths_violin, ax_contour_lengths_violin = my_plots[\"contour_lengths\"][\"violin\"].values()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fb3ac863-85ba-4d9b-b366-369463c194a5",
+   "metadata": {},
+   "source": [
+    "We can now look at the figure and modify these as we would like, for example to change the `title` and `xlabel` values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc8bc1ad-c388-4817-9b3f-651cc26291a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ax_area_dist.set_title(\"This is the Area of minicircle grains\")\n",
+    "ax_area_dist.set_xlabel(\"Area in nanometres squared\")\n",
```

Comment:
I don't see these changes when I run this cell (might be related to above comment):
![Screenshot 2023-03-06 at 11 19 12](https://user-images.githubusercontent.com/91465918/223096162-00f6e6a7-5aa4-414a-927f-cd916ee359e0.png)

Reply:
Is it a cell scrolling issues? The image may be hidden perhaps. Works here...

![image](https://user-images.githubusercontent.com/17566406/223122925-8c2df9a4-1b18-4212-8347-1c6d8dd4f6b1.png)

Reply:
For some reason I can't scroll but saving it produces the expected